ci_config.json

```diff
@@ -414,6 +414,11 @@
       "libjack-dev"
     ]
   },
+  "sdl2": {
```

Comment:
I’d get rid of this honestly.

Reply:
yeah, Its just there to verify the CI works, I'll remove it as soon as we know this is solid.

Reply:
Seems fine now.